scripts/update-monitoring-mixin-dashboards.sh

```diff
@@ -11,33 +11,95 @@
 
 set -e
 
-TMPDIR="$(mktemp -d)"
-trap "rm -rf $TMPDIR" EXIT
+GRIZZLY_VERSION="0.4.3"
+GRIZZLY_BIN="grr"
 
+JB_VERSION="0.5.1"
+JB_BIN="jb"
 
-main() {
+YQ_VERSION="4.44.3"
+YQ_BIN="yq"
+
+SCRIPT_DIR=$(cd $(dirname "${BASH_SOURCE}") && pwd -P)
+OS=$(go env GOOS)
+ARCH=$(go env GOARCH)
+TMPDIR="$(mktemp -d -t giantswarm-dashboards-XXXXXX)"
+#trap "rm -rf $TMPDIR" EXIT
+echo $TMPDIR
+
+function install_tool() {
```

Comment:
I feel like we use this quite a lot ( in prometheus rules for instance) is there no way to reuse the same things?

Reply:
There's no easy way to reuse this AFAIK

Reply:
Can we not git clone a script we have somewhere else?

Reply:
We could fetch something from github but I am not sure I want to got this direction.

Reply:
I understand but then maybe we should use a container image for this instead of downloading tools locally?

Reply:
We could but do not want to add this now.